searchpress.php

```diff
@@ -34,57 +34,57 @@
 
 // To communicate with the ES API.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-api.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-api.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 // Settings, mappings, etc. for ES.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-config.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-config.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 // Heartbeat.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-heartbeat.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-heartbeat.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 // An object wrapper that becomes the indexed ES documents.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-post.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-post.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 // A controller for syncing content across to ES.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-sync-manager.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-sync-manager.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 // Manages all cron processes.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-cron.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-cron.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 // Manages metadata for the syncing process.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-sync-meta.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-sync-meta.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 // You know, for search.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-search.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-search.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 // Extends the search with WP-style arguments.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-wp-search.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-wp-search.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 // Replaces core search with SearchPress.
-require_once SP_PLUGIN_DIR . '/lib/class-sp-integration.php';
+require_once SP_PLUGIN_DIR . '/lib/class-sp-integration.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 
 if ( is_admin() ) {
-	require_once SP_PLUGIN_DIR . '/lib/class-sp-admin.php';
+	require_once SP_PLUGIN_DIR . '/lib/class-sp-admin.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 }
 
 if ( defined( 'WP_CLI' ) && WP_CLI ) {
-	include SP_PLUGIN_DIR . '/bin/wp-cli.php';
+	include SP_PLUGIN_DIR . '/bin/wp-cli.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
 }
 
 if ( defined( 'SP_DEBUG' ) && SP_DEBUG ) {
-	include SP_PLUGIN_DIR . '/lib/class-sp-debug.php';
+	include SP_PLUGIN_DIR . '/lib/class-sp-debug.php'; // phpcs:ignore WordPressVIPMinimum.Files.IncludingFile.IncludingFile
```

Comment:
phpcs supports enable/disable like eslint. that would be better here, rather than repeating it like 20 times

Reply:
At the file level? That makes sense. I'll do that.

Reply:
I meant using `phpcs:disable WordPressVIPMinimum.Files.IncludingFile.IncludingFile` and `phpcs:enable WordPressVIPMinimum.Files.IncludingFile.IncludingFile`, but at the file level works too.